packages/core/integration-tests/test/html.js

```diff
@@ -312,6 +316,11 @@ describe('html', function () {
     );
     assert(html.includes(`<meta name="msapplication-config" content="none">`));
     assert(html.includes(`<meta property="og:image" content="/logo.svg">`));
+    assert(
+      html.includes(
+        `<meta name="msapplication-TileImage" content="/logo.14afa1de.svg">`,
```

Comment:
This has to be a regex check instead `/<meta name="msapplication-TileImage" content="/logo\.[0-9a-f]+\.svg">`/`